lib/assets/javascripts/manage-prototype/manage-plugins.js

```diff
@@ -0,0 +1,110 @@
+window.GOVUKPrototypeKit.documentReady(() => {
+  const params = new URLSearchParams(window.location.search)
+  const packageName = params.get('package')
+  const mode = params.get('mode') || window.location.pathname.split('/').pop()
+  let startTimestamp = ''
+  let requestTimeoutId
+  let timedOut = false
+  const timeout = 30 * 1000
+  const token = document.querySelector('meta[name="csrf-token"]').getAttribute('content')
+
+  const show = (id) => document.getElementById(id)
+    .classList.remove('govuk-!-display-none')
+
+  const hide = (id) => document.getElementById(id)
+    .classList.add('govuk-!-display-none')
+
+  const showCompleteStatus = () => {
+    clearTimeout(actionTimeoutId)
+    hide('panel-processing')
+    show('panel-complete')
+    show('instructions-complete')
+  }
+
+  const showErrorStatus = () => {
+    hide('panel-processing')
+    show('panel-error')
+  }
+
+  const postRequest = (url) => fetch(url, {
+    method: 'POST',
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+      'CSRF-Token': token
+    },
+    body: JSON.stringify({
+      package: packageName
+    })
+  })
+
+  const actionTimeoutId = setTimeout(() => {
+    timedOut = true
+  }, timeout)
+
+  const pollStatus = (timeout) => {
+    if (requestTimeoutId) {
+      clearTimeout(requestTimeoutId)
+    }
+    if (timedOut) {
+      showErrorStatus()
+    } else {
+      postRequest('/manage-prototype/plugins/status')
+        .then(response => response.json())
+        .then(data => {
+          requestTimeoutId = setTimeout(() => {
+            if (data.status === 'error') {
+              clearTimeout(actionTimeoutId)
+              showErrorStatus()
+            } else if (data.startTimestamp === startTimestamp) {
+              // poll status again if prototype hasn't restarted
+              pollStatus(timeout - 1000)
+            } else {
+              showCompleteStatus()
```

Comment:
Some `console.log` statements for debugging might be useful for any future issues that may crop up...

Reply:
Added some console logging as requested